crypto/x509v3/v3_alt.c

```diff
@@ -67,11 +67,33 @@ STACK_OF(CONF_VALUE) *i2v_GENERAL_NAME(X509V3_EXT_METHOD *method,
 {
     unsigned char *p;
     char oline[256], htmp[5];
-    int i;
+    int i, nid;
     switch (gen->type) {
     case GEN_OTHERNAME:
-        if (!X509V3_add_value("othername", "<unsupported>", &ret))
-            return NULL;
+        nid = OBJ_obj2nid(gen->d.otherName->type_id);
+        if (nid != NID_undef) {
```

Comment:
I think this would be more useful if it didn't require that the OID had a NID and instead used the numerical form of the OID if no NID was present.